utils/build-windows-toolchain.bat

```diff
@@ -635,6 +635,7 @@ cmake ^
   -D SwiftPM_DIR=%BuildRoot%\14\cmake\modules ^
   -D IndexStoreDB_DIR=%BuildRoot%\15\cmake\modules ^
   -D SwiftCollections_DIR=%BuildRoot%\13\cmake\modules ^
+  -D SwiftSyntax_DIR=%BuildRoot%\13\cmake\modules ^
```

Comment:
I don’t think this will work. We should add `SwiftSyntax` as project 16 and renumber SK-LSP to 17. Now you’re falsely pointing it to Swift Collection.